packages/@aws-cdk/aws-glue-alpha/test/database.test.ts

```diff
@@ -88,6 +111,14 @@ test('locationUri length must be <= 1024', () => {
   ).toThrow();
 });
 
+test('description length must be <= 2048', () => {
+  expect(() =>
+    new glue.Database(stack, 'Database', {
+      description: 'a'.repeat(2049),
+    }),
+  ).toThrow();
```

Comment:
```suggestion
  ).toThrow(\description length must be less than or equal to 2048, got 2049\);
```
nit: add error message to test.